web/src/components/Menu/Menu.module.css

```diff
@@ -64,4 +84,15 @@ a {
     text-decoration: none;
     letter-spacing: 0.1em;
     color: #344FAB;
+}
+
+.menu-closed {
+    border: none;
+    background-color: transparent;
+}
+
+
+.menu-open {
+    border: none;
+    background-color: transparent;
```

Comment:
Molaria posarli alguna transició quan es fa hover, podriem comentarli a l'Adriana a ver que podriem fer

Reply:
I like the plan, m'invento un per ara